test/tests/orbit/unit/updatable-test.js

```diff
@@ -0,0 +1,221 @@
+import { successfulOperation, failedOperation } from 'tests/test-helper';
+import Updatable from 'orbit/updatable';
+import Transform from 'orbit/transform';
+import { Promise } from 'rsvp';
+
+var Source, source;
+
+module('Orbit - Updatable', {
+  setup: function() {
+    source = {};
+    Updatable.extend(source);
+  },
+
+  teardown: function() {
+    Source = source = null;
+  }
+});
+
+test('it exists', function(assert) {
+  assert.ok(source);
+});
+
+test('it should mixin Updatable', function(assert) {
+  assert.ok(source._updatable, 'should have `_updatable` flag');
+});
+
+test('it should resolve as a failure when _update fails', function(assert) {
+  assert.expect(2);
+
+  source._update = function(query) {
+    return failedOperation();
+  };
+
+  stop();
```

Comment:
Good call. I've been trying to return promises as much as possible as a clean alternative.